src/lib/nav_store.ts

```diff
@@ -0,0 +1,9 @@
+import type { Writable } from 'svelte/store'
+import { writable } from 'svelte/store'
+
+const route: Writable<{ hash: string; pathname: string }> = writable({
```

Comment:
`Writable` accepts a generic, which lets us specify the type of thing we are storing.